src/components/ListBusinesses.js

```diff
@@ -41,10 +42,10 @@ const ListBusinesses = () => {
       </tr> */}
           {business.map((listing) => (
             <tr>
-              <td>{listing.name}</td>
+              <td>{listing.bname}</td>
               <td>{listing.city}</td>
               <td>{listing.country}</td>
-              <td>{listing.lift}</td>
+              <td>{listing.lift ? "✅" : "❎"}</td>
```

Comment:
I like these! Just a tip: for emojis and symbols, I got complaints from Prettier. If you happen to run into the same issues, you can find the UTF-8 equivalents for these symbols [here](https://www.w3schools.com/charsets/ref_utf_dingbats.asp)! If you don't run into issues, nvm! 

WHITE HEAVY CHECK MARK
https://www.w3schools.com/charsets/tryit.asp?deci=9989
NEGATIVE SQUARED CROSS MARK
https://www.w3schools.com/charsets/tryit.asp?deci=10062